### PR TITLE
Added Comments and Adjusted Skills.simba

### PR DESCRIPTION
**Pull request by StickToTheScript**
Created TSkill and TSkillIndex types for use with Skills.simba

### CODE REVIEW

File: Reflection/Timing.simba [hunk 1]
```diff
@@ -1,34 +1,76 @@
...
-Procedure Timer.Start;
+{*
+R_TileToMM
+~~~~~~~~~~
+Converts the input tile to a point on the minimap.
+
+Example:
+  Mouse.Move(R_TileToMM(R_GetMe.Tile));
+*}
+Function R_TileToMM(Tile: TPoint): TPoint;
```
Comment (Brandon-T) [inline Reflection/Timing.simba]: We are converting a `Tile` to MM right? Not `MSToMM` (point to MM)?

```suggestion
Function R_TileToMM(Tile: RSTile): TPoint;
```

File: Reflection/Timing.simba [hunk 2]
```diff
@@ -1,34 +1,76 @@
...
+Returns the height of a given tile.
+
+Example:
+  if (R_TileHeight > 13) then
+    ...
+*}
+Function R_TileHeight(Tile: TPoint): Int32;
 begin
-  self.Tick := 0;
+  Result := RSTile.GetHeight;
```
Comment (Brandon-T) [inline Reflection/Timing.simba]: This is a bug. `RSTile` is not defined here. The parameter is `Tile`. It should be an `RSTile` as the parameter?

File: Reflection/Timing.simba [hunk 3]
```diff
@@ -1,34 +1,76 @@
...
-Function R_WaitFunction(timeout: UInt32; timePerLoop: Int32 = 50; func: Function: Boolean): Boolean;
+{*
+R_TileBounds
+~~~~~~~~~~~~
+Returns the mainscreen bounds of a given tile.
+
+Example:
+  TileBounds := R_TileBounds(R_GetMe.Tile);
+*}
+Function R_TileBounds(Tile: TPoint): TRectangle;
```
Comment (Brandon-T) [inline Reflection/Timing.simba]: Should be:
```suggestion
Function R_TileBounds(Tile: RSTile): TRectangle;
```

**Review by Brandon-T — CHANGES_REQUESTED** [review]

File: Reflection/Timing.simba (hunk 2 above)
Comment (StickToTheScript) [reply Reflection/Timing.simba]: Updated and fixed.

**Review by Brandon-T — APPROVED** [review]

**Pull request merged**